migrations/2024-11-21-195004_add_report_count/up.sql

```diff
@@ -0,0 +1,79 @@
+-- Adding report_count and unresolved_report_count
+-- to the post and comment aggregate tables
+ALTER TABLE post_aggregates
+    ADD COLUMN report_count bigint NOT NULL DEFAULT 0,
+    ADD COLUMN unresolved_report_count bigint NOT NULL DEFAULT 0;
```

Comment:
bigint seems complete overkill, I cant imagine ever having any post with 9.223.372.036.854.775.807 (i64::MAX) reports.

Reply:
Good point, should I do smallint / int2 / i16 ? https://kotiri.com/2018/01/31/postgresql-diesel-rust-types.html

Reply:
Yes that sounds good.